.github/workflows/test.yaml

```diff
@@ -47,7 +44,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           persist-credentials: false
-      - run: apt install valgrind
+      - name: Install valgrind
+        run: sudo apt-get install -y valgrind
       - run: export VALGRIND="valgrind -q"
       - run: make test
```

Comment:
Interesting. What's the mechanism here that makes this run with valgrind turned on? Does Rust look for the VALGRIND env var?

Reply:
Nope, test.sh is supposed to check for it! But I messed up the logic in test.sh; fixed in the latest push.